source/js/schemes/pisces.js

```diff
@@ -1,132 +1,85 @@
-      let scrollHeight = Math.max($(document).height(), $(document.body).height());
-      if (typeof offset !== 'object') offsetBottom = offsetTop = offset;
-      if (typeof offsetTop === 'function') offsetTop = offset.top(this.$element);
-      if (typeof offsetBottom === 'function') offsetBottom = offset.bottom(this.$element);
-      let affix = this.getState(scrollHeight, height, offsetTop, offsetBottom);
-      if (this.affixed !== affix) {
-        if (this.unpin != null) this.$element.css('top', '');
-        let affixType = 'affix' + (affix ? '-' + affix : '');
-        let e = new $.Event(affixType + '.bs.affix');
-        this.$element.trigger(e);
-        if (e.isDefaultPrevented()) return;
-        this.affixed = affix;
-        this.unpin = affix === 'bottom' ? this.getPinnedOffset() : null;
-        this.$element
-          .removeClass(Affix.RESET)
-          .addClass(affixType)
-          .trigger(affixType.replace('affix', 'affixed') + '.bs.affix');
-      }
-      if (affix === 'bottom') {
-        this.$element.offset({
-          top: scrollHeight - height - offsetBottom
-        });
-      }
+    if (affix === 'bottom') {
+      $(this.element).offset({
+        top: scrollHeight - height - offsetBottom
+      });
     }
   }
-
-  Affix.RESET = 'affix affix-top affix-bottom';
-
-  function Plugin(option) {
-    return this.each(function() {
-      let $this = $(this);
-      let data = $this.data('bs.affix');
-      let options = typeof option === 'object' && option;
-      if (!data) $this.data('bs.affix', data = new Affix(this, options));
-      if (typeof option === 'string') data[option]();
-    });
-  }
-
-  $.fn.affix = Plugin;
-  $.fn.affix.Constructor = Affix;
-})(jQuery);
+};
 
 window.addEventListener('DOMContentLoaded', () => {
   const sidebarOffset = CONFIG.sidebar.offset || 12;
-  const sidebarInner = document.querySelector('.sidebar-inner');
-
-  const getHeaderOffset = () => document.querySelector('.header-inner').offsetHeight + sidebarOffset;
 
-  const getFooterOffset = () => {
-    let footer = document.querySelector('#footer');
-    let footerInner = document.querySelector('.footer-inner');
-    let footerMargin = footer.offsetHeight - footerInner.offsetHeight;
-    let footerOffset = footer.offsetHeight + footerMargin;
-    return footerOffset;
-  };
+  let headerOffset = document.querySelector('.header-inner').offsetHeight + sidebarOffset;
+  let footer = document.querySelector('#footer');
+  let footerInner = document.querySelector('.footer-inner');
+  let footerMargin = footer.offsetHeight - footerInner.offsetHeight;
+  let footerOffset = footer.offsetHeight + footerMargin;
 
-  const initAffix = () => {
-    let headerOffset = getHeaderOffset();
-    let footerOffset = getFooterOffset();
-    $('.sidebar-inner').affix({
-      offset: {
-        top   : headerOffset - sidebarOffset,
-        bottom: footerOffset
-      }
-    });
-    document.querySelector('#sidebar').css({
```

Comment:
Can ID may be faster to find than class?

Reply:
Good question, I have not investigated it yet, but I believe this is much more efficient than jquery.

Reply:
I'm not about Jquery, this must work in CSS.